sql/mob_spawn_points.sql

```diff
@@ -58242,11 +58247,11 @@ INSERT INTO `mob_spawn_points` VALUES (17658244,'Hannequet','Hannequet',10200,46
 INSERT INTO `mob_spawn_points` VALUES (17658245,'Hannequet','Hannequet',10200,466.629,19.987,-32.151,40);
 INSERT INTO `mob_spawn_points` VALUES (17658246,'Hannequet','Hannequet',10200,472.575,19.572,-39.897,70);
 INSERT INTO `mob_spawn_points` VALUES (17658247,'Ephemeral_Murex','Ephemeral Murex',10193,439.16,20.968,21.467,100);
-INSERT INTO `mob_spawn_points` VALUES (17658248,'Tejas','Tejas',10230,0,0,0,130);
-INSERT INTO `mob_spawn_points` VALUES (17658249,'Tejas','Tejas',10230,0,0,0,160);
-INSERT INTO `mob_spawn_points` VALUES (17658250,'Tejas','Tejas',10230,0,0,0,190);
-INSERT INTO `mob_spawn_points` VALUES (17658251,'Tejas','Tejas',10230,0,0,0,220);
-INSERT INTO `mob_spawn_points` VALUES (17658252,'Tejas','Tejas',10230,0,0,0,250);
+INSERT INTO `mob_spawn_points` VALUES (17658248,'Tejas','Tejas',0,-544,-3,114,12);
```

Comment:
Main problem is that those mobs either have multiple spawns or move (and also the fact that back then, plogger has no hex to float conversion) -> old packet from 2013 with converted hex pos for Gieremund
[01:55:50] 0x0E, 68 Bytes - (Update) Entity
   0  1  2  3  4  5  6  7  8  9  A  B  C  D  E  F
   -----------------------------------------------
   0E 22 74 02 B0 71 0D 01 B0 01 0F 40 33 F3 D1 C3    ."t.°q..°..@3óÑÃ
   12 83 90 C0 17 79 91 43 AD 08 00 00 28 28 00 03    .ƒÀ.y‘C­...((..
   07 9D 00 00 00 1C 00 20 20 00 08 00 8D 65 01 00    ......  ...e..
   00 00 6C 01 47 69 65 72 65 6D 75 6E 64 00 00 00    ..l.Gieremund...
   00 00 00 00                                        ....

-419.9 -4.516 290.964